Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -88,6 +88,79 @@
       countdown_sound:
         path: /Audio/Effects/minibombcountdown.ogg
 
+- type: entity
+  name: supermatter grenade
+  description: Grenade that simulates delamination of supermatter engine, pulling things in heap and explode after some time.
+  parent: GrenadeBase
+  id: SupermatterGrenade
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Grenades/supermattergrenade.rsi
+  - type: OnUseTimerTrigger
+    delay: 3
+  - type: Explosive
+    explosionType: Default
+    totalIntensity: 200
+    intensitySlope: 30
+    maxIntensity: 120
+  - type: Appearance
+    visuals:
+    - type: TimerTriggerVisualizer
+      countdown_sound:
+        path: /Audio/Effects/minibombcountdown.ogg
+  - type: SingularityDistortion
+    intensity: 0
+    falloffPower: 2
+  - type: GravityWell
+    maxRange: 8
+    baseRadialAcceleration: 0
+    gravPulsePeriod: 0.01
+  - type: AmbientSound
+    enabled: False
+    volume: -5
+    range: 14
+    sound:
+      path: /Audio/Effects/Grenades/Supermatter_Loop.ogg
+  - type: PointLight
+    enabled: false
+    color: "#ffaa44"
+    energy: 8
+    radius: 6
+    softness: 1
+    offset: "0, 0"
+  - type: SupermatterGrenade
+    explodeAfterGravityPull: True
+    timeTillExplosion: 8
+    distortionIntensity: 100
+    baseRadialAcceleration: 145
+    gravityPullStartSoundVolume: 10
+    gravityPullEndSoundVolume: 5
+    gravityPullLoopSoundOffset: 0.6
+
+- type: entity
+  name: whitehole grenade
```

Comment:
Maybe name it 'inverse supermatter grenade' to relate it to the other one?

Reply:
Too long... whitehole is more informative